utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -944,6 +951,29 @@ int main(int argc, char **argv)
 
                     break;
                 }
+                case 79:
+                {
+                    testCaseName = "remap";
+
+                    if (interpolationType != RpptInterpolationType::NEAREST_NEIGHBOR && interpolationType != RpptInterpolationType::BILINEAR)
```

Comment:
We should be checking this inside the API and return unsupported, not in the test suite

Reply:
This was added for rotate so added the same here too.